code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -21,18 +22,18 @@
 	if(!O) return
 
 	if(num_fragments)
-		var/lying = FALSE
+		var/ontop = FALSE
 		if(isturf(src.loc))
 			for(var/mob/living/M in O)
-				//lying on a frag grenade while the grenade is on the ground causes you to absorb most of the shrapnel.
-				//you will most likely be dead, but others nearby will be spared the fragments that hit you instead.
-				if(M.lying)
-					lying = TRUE
+				ontop = TRUE
```

Comment:
You do not understand the consequences of this change. It will make grenade launchers instakill anyone hit.
This was meant to be only for lying for a reason.

Reply:
Hm , you're right , im gonna look into making grenades bounce off people if they fall ontop of them.